### ISSUE TITLE
flutter line sdk-1.3.0 'flutter line sdk/flutter line sdk-Swift.h' file not found

### ISSUE THREAD

**Issue by JW-chenjingwei** [line/flutter_line_sdk#61]
sdk版本：line sdk-1.3.0

看了文档要求使用use_frameworks!，但是项目已经被禁止使用use_frameworks!，改用了use_modular_headers!，所以会一直报这个错误，
建议：
在FlutterLineSdkPlugin.m文件更改：
```
#import "FlutterLineSdkPlugin.h"

#if __has_include( <flutter_line_sdk/flutter_line_sdk-Swift.h>)
#import <flutter_line_sdk/flutter_line_sdk-Swift.h>
#else
// Support project import fallback if the generated compatibility header
// is not copied when this plugin is created as a library.
// https://forums.swift.org/t/swift-static-libraries-dont-copy-generated-objective-c-header/19816
#import "flutter_line_sdk-Swift.h"

#endif
@implementation FlutterLineSdkPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
  [SwiftFlutterLineSdkPlugin registerWithRegistrar:registrar];
}
@end

```

可以更新个小版本吗？

**Comment by onevcat** [comment]
Oh. I will check and see if it is a workable solution. Thanks for the feedback!

**Comment by onevcat** [comment]
Should be solved in #62 and I will create a tag soon. Now the package should also support built as static library.

**Comment by onevcat** [comment]
Version 2.3.0 was released with support for the static library. For any existing users, this change should not cause any issue and the upgrade should be transparent.

https://pub.dev/packages/flutter_line_sdk/versions/2.3.0
